pkg/translator/prometheusremotewrite/metrics_to_prw_v2.go

```diff
@@ -80,7 +80,12 @@ func (c *prometheusConverterV2) fromMetrics(md pmetric.Metrics, settings Setting
 					}
 					c.addGaugeNumberDataPoints(dataPoints, resource, settings, promName)
 				case pmetric.MetricTypeSum:
-					// TODO implement
+					dataPoints := metric.Sum().DataPoints()
+					if dataPoints.Len() == 0 {
+						errs = multierr.Append(errs, fmt.Errorf("empty data points. %s is dropped", metric.Name()))
```

Comment:
@dashpole you did a [comment here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36353/files#r1842392516) saying to remove the error line. Should I remove it?

Reply:
Yes, but please remove it from the gauge case above as well

Reply:
[6c78f84](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37156/commits/6c78f84b4ddd902658480c2ff71c5be599aea198)

Removed